platform/accessibility/common-issues/pdf/table-missing-headers.md

```diff
@@ -0,0 +1,76 @@
+# [PDF]: Table is missing headers
+
+## Table of contents
+
+  - [Descrption of the issue](#description-of-the-issue)
+  - [Definition of done](#definition-of-done)
+  - [Steps to fix the issue](#steps-to-fix-the-issue)
+    - [Define table headers in Microsoft Word](#define-table-headers-in-microsoft-word)
+    - [Define table headers in Acrobat Pro](#define-table-headers-in-acrobat-pro)
+  - [Test for compliance](#test-for-compliance)
+    - [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+    - [Acrobat Pro accessibility tools](#acrobat-pro-accessibility-tools)
+  - [Learn more](#learn-more)
+
+## Description of the issue
+
+- Synopsis: Make tables accessible in PDF documents by using table elements. The applicable standard from Section508.22.h dictates to "associate data and header cells for multi-dimensional data tables."
+- Severity: Critical
+- Deque University help link: https://dequeuniversity.com/rules/comply-pdf/1.0/table-miss-header
+- WCAG Success Criteria: [1.3.1 Info and Relationships](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
+
+## Definition of done
+
+- There SHOULD be **0 table header violations** in the Word [Check Accessibility panel](https://support.microsoft.com/en-us/office/improve-accessibility-with-the-accessibility-checker-a16f6de0-2f39-4a2b-8bd8-5ad801426c7f)
+- There MUST be  in the Acrobat [Accessibility Checker](https://www.adobe.com/accessibility/products/acrobat/using-acrobat-pro-accessibility-checker.html)
+- Consider exporting an accessibility report from Acrobat while running the Accessibility Checker
+
+## Steps to fix the issue
+
+Tables that are missing table headers can be remediated in Word or Acrobat. The preferred method illustrated here is to fix the issue in a Word document whenever possible. Instructions to fix a table header in Acrobat are included in the second documentation link.
```

Comment:
If you want me to get nit-picky about details, I would change:
"Tables that are missing table headers can be remediated in Word or Acrobat. The preferred method illustrated here is to fix the issue in a Word document whenever possible. Instructions to fix a table header in Acrobat are included in the second documentation link."
to
"Remediate tables in Word (preferred) or Acrobat."

Reply:
Also, this made me think that I should add something to the word list about how to refer to Word and Acrobat. (e.g., Microsoft Word vs. Word and Adobe Acrobat vs. Acrobat) No need to change anything in your doc at this time, but I'll follow up with you if we add guidance to the word list for this.

Reply:
Good call. I'll go ahead and add the company names because I've heard some questions about "Adobe" when I think the ask was about "Adobe Acrobat".